img/foxsi_image_map.pro

```diff
@@ -75,7 +74,7 @@ default, fov, 20.
 	image = temp
 
 	map = make_map( image, dx=stripsize, dy=stripsize, xcen=xc, ycen=yc, $
-		time=time, id='Det'+strtrim(detnum,2), fov=2*fov )
+		time=time, id='Det'+strtrim(detnum,2), fov=2*fov, dur=trange[1]-trange[0] )
```

Comment:
Does using the keyword "dur" just save the duration in the map structure, or does it serve an additional function? This is just for my own understanding!